docs/sources/whatsnew/whats-new-in-v9-4.md

```diff
@@ -0,0 +1,208 @@
+
+Use the keyboard shortcut cmd + K on Mac or ctrl + K on Linux/Windows to launch the command palette.
+
+Read more about using the command palette [in the documentation.](https://grafana.com/docs/grafana/latest/search/)
+
+{{< figure src="/static/img/docs/navigation/command-palette-9-4.gif" max-width="750px" caption="Grafana command palette" >}}
+
+## New navigation
+
+Early access on Grafana Cloud
+
+Grafana’s navigation has been reorganized to make it easier for you to access the data you need. With this update, you'll be able to quickly navigate between features, giving you full visibility into the health of your systems.
+
+The new navigation is being gradually rolled out to users on Grafana Cloud.
+
+_Note:_ The Grafana documentation has not yet been updated to reflect changes to the navigation.
+
+{{< figure src="/static/img/docs/navigation/navigation-9-4.png" max-width="750px" caption="Grafana new navigation" >}}
+
+## Auditing and Usage Insights: Support for Loki multi-tenancy
+
+_This feature is available for Enterprise customers_
+This feature adds support to push analytics events and auditing logs to Loki with multi-tenancy mode, by specifying a tenant id.
+
+## Reporting: Enable changing the report scale factor
+
+_This feature is available for Enterprise customers_
+Scale factor is a new feature for reports that allows users to change the dimension of the panels of the PDF document. It allows you to show more columns in the tables zooming out or show panels bigger zooming in.
+You can modify the scale factor for each report in the report editor and/or when you share the whole PDF directly from the dashboard page.
+
+{{< figure src="/media/docs/grafana/FormatReportScheduler9.4.png" max-width="750px" caption="Scale factor feature in Report format page" >}}
+
+{{< figure src="/media/docs/grafana/FormatReportShare9.4.png" max-width="750px" caption="Scale factor feature in Share functionality" >}}
+
+## Dashboard panel redesign
+
+Dashboard panels hold a lot of information, some of which is difficult to discover or access entirely from the dashboard itself. With our redesigned panels, we've improved accessibility and made it easier to understand the status of a panel by adding and moving key elements.
+
+We’ve rethought the panel information architecture, added additional interaction points, and reduced visual clutter. To start, we’ve improved the support of panels without a header, made a distinction between details set by you and data-induced information, and then included all essential components in the header of the panel. All of these are laid out from left to right in a row, so there are no overlapping, unusable components.
+
+Grafana’s new panel is available only for React-based panels; no Angular-based panels will have the redesign (i.e., they’ll have the old Graph and Table visualizations).
+
+{{< video-embed src="/media/docs/grafana/screen-recording-panel-header-redesign-whats-new-9-4.mp4>" >}}
```

Comment:
We've added a view as it feels the new panel header is hard to grasp without it, but I understand it might be too heavy. Let us know if we should remove it or find something else.